consumer/config.go

```diff
@@ -0,0 +1,52 @@
+package consumerLibrary
+
+import "sync"
+
+type LogHubConfig struct {
+	//:param endpoint:
+	//:param access_key_id:
+	//:param access_key:
+	//:param project:
+	//:param logstore:
+	//:param consumer_group_name:
+	//:param consumer_name:
+	//:param cursor_position: This options is used for initialization, will be ignored once consumer group is created and each shard has beeen started to be consumed.
+	//:param heartbeat_interval: default 20, once a client doesn't report to server * heartbeat_interval * 2 interval, server will consider it's offline and re-assign its task to another consumer. thus  don't set the heatbeat interval too small when the network badwidth or performance of consumtion is not so good.
+	//:param data_fetch_interval: default 2, don't configure it too small (<1s)
```

Comment:
//:param data_fetch_interval: default 2 (seconds), don't configure it too small (<1s)